wgengine/router/router_linux.go

```diff
@@ -62,6 +62,9 @@ type linuxRouter struct {
 
 	nfr linuxfw.NetfilterRunner
 	cmd commandRunner
+
+	magicsockPort4 uint16
+	magicsockPort6 uint16
```

Comment:
NIT: Just to be more explicit, could these be `magicsockPortV4` and `magicsockPortV6`?